README.md

```diff
@@ -1,33 +1,123 @@
 # What is this working group about?
-This Working Group focused on Growth-Maturity-Decline metrics and software.
 
-The goal is to refine the metrics that inform [Growth-Maturity-Decline][gmd] and to work with software implementations.
+This is a [CHAOSS](https://chaoss.community) working group, 
+focused on the [Growth-Maturity-Decline][gmd] metrics category.
+Its aim is to coordinate the definition of the metrics, and
+the production of software for implementing them, in this area.
 
-[gmd]: https://github.com/chaoss/metrics/blob/master/2_Growth-Maturity-Decline.md
+[gmd]: gmd_metrics.md
 
-## Meetings
-Every other Wednesday at 11am CDT / 18:00 CET. ([local time](http://www.thetimezoneconverter.com/?t=11am&tz=Chicago&)) in the CHAOSS Zoom room https://unomaha.zoom.us/j/720431288 (Meeting ID for [dial in](https://unomaha.zoom.us/zoomconference?m=DKGo2mmIuOv9xSjphoGZZmYKxr5HFrS9): 720 431 288)
+# How to participate
+
+You can start by introducing yourself on the CHAOSS mailing list (see below) explaining your interest.
+Then, you can have a look at the archives of the mailing lists, at the minutes of past meetings,
+and at the [issues](https://github.com/chaoss/wg-gmd/issues) and
+[pull requests](https://github.com/chaoss/wg-gmd/pulls) in
+[this repository](https://github.com/chaoss/wg-gmd).
+
+Then, of course you can participate in the mailing list,
+in online meetings, and in issues and pull requests.
+
+Currently, main lines of work are:
+
+* [Use cases](use_cases). Proposal and discussion of use cases that help to understand metrics in context.
```

Comment:
I think the relationship of the use cases ot goals, questions metrics needs to be more clear.

Reply:
Right now there is little relationship described, since we still have little experience on how it will work... Would you mind that we start with this text for now, and try to refine in further pull requests.?

Reply:
yes. Lets go with this for now!